File: app/src/main/java/com/panic/sign/PanicSign.java
package com.panic.sign;

import android.app.Application;
import android.content.Context;

public class PanicSign extends Application {
    private static Context context;

    @Override
    public void onCreate() {
        super.onCreate();
        context = this.getApplicationContext();
    }

    static Context getAppContext() {
        return context;
    }
}

File: app/src/main/java/de/bastianrinsche/panicsign/PanicSign.kt
package de.bastianrinsche.panicsign

import android.app.Application

class PanicSign : Application() {
    override fun onCreate() {
        super.onCreate()
        colorUtils = ColorUtils(applicationContext)
    }

    companion object {
        lateinit var colorUtils: ColorUtils
        val signService by lazy {
            SignService.create()
        }
    }
}
